webapp/app/utils/createReducer.js

```diff
@@ -0,0 +1,9 @@
+export const createReducer = (initialState, handlers) => {
+  return function reducer(state = initialState, action) {
+    if (handlers.hasOwnProperty(action.type)) {
+      return handlers[action.type](state, action);
+    }
+
+    return state;
+  };
+};
```

Comment:
You Implemented by hand what handleActions Does

Reply:
Not sure Im following what by hand means... we type everything, please provide a clear example